server/integration/jgroups/src/main/java/org/infinispan/server/jgroups/JChannelFactory.java

```diff
@@ -173,16 +163,16 @@ public Object handleUnknownForkChannel(Message message, String forkChannelId) {
             }
 
             private Object handle(Message message) {
-                Header header = (Header) message.getHeader(this.id);
+                Header header = message.getHeader(this.id);
                 // If this is a request expecting a response, don't leave the requester hanging - send an identifiable response on which it can filter
                 if ((header != null) && (header.type == Header.REQ) && header.rspExpected()) {
-                    Message response = message.makeReply().setFlag(message.getFlags()).clearFlag(Message.Flag.RSVP, Message.Flag.SCOPED);
+                    Message response = message.makeReply().setFlag(message.getFlags()).clearFlag(Message.Flag.RSVP);
```

Comment:
The JGroups subsystem doesn't have a dependency on core, so they aren't available here.